### PR TITLE
API Updates for beta branch

### PR DESCRIPTION
**Pull request by pakrym-stripe**
Codegen for openapi v167.
r? @dcr-stripe
cc @stripe/api-libraries

## Changelog
- Updated stable APIs to the latest version
- Add `QuotePhase` resource.

### CODE REVIEW

File: src/main/java/com/stripe/model/Quote.java [hunk 1]
```diff
@@ -940,193 +940,6 @@ public static class InvoiceSettings extends StripeObject {
     Long daysUntilDue;
   }
 
-  @Getter
-  @Setter
-  @EqualsAndHashCode(callSuper = false)
-  public static class QuotePhaseConfiguration extends StripeObject implements HasId {
```
Comment (dcr-stripe) [inline src/main/java/com/stripe/model/Quote.java:946]: Double checking we haven't published a beta version with this embedded class right?

**Review by dcr-stripe — APPROVED** [review]
LGTM!

**Review by dcr-stripe — CHANGES_REQUESTED** [review]
un-LGTM.

Why does `QuotePhaseRetrieveParams` have pagination params?

File: src/main/java/com/stripe/model/Quote.java (hunk 1 above)
Comment (pakrym-stripe) [reply src/main/java/com/stripe/model/Quote.java:946]: Nope.

**Pull request closed without merging**